SPECS/opensp/opensp.spec

```diff
@@ -1,25 +1,23 @@
 %define _module_build 1
```

Comment:
Small request: since `_module_build` seems to be hard-coded and we don't manipulate such macros through our build tools, could you remove the macro declaration while we're here and update the rest of the spec accordingly to clean it up a little?